src/test/java/unstable/MenuControlTest.java

```diff
@@ -150,14 +159,31 @@ public void menuControlTest() {
         // Testing board save
         press(CLOSE_PANEL);
         click("Boards");
-        push(KeyCode.DOWN);
+        push(KeyCode.DOWN).push(KeyCode.DOWN);
         push(KeyCode.ENTER);
         PlatformEx.waitOnFxThread();
         assertEquals(2, panelControl.getNumberOfSavedBoards());
-        
+
+        // Testing board create new and confirm
+        click("Boards");
+        push(KeyCode.DOWN).push(KeyCode.ENTER);
+        click("OK");
+        assertEquals(0, panelControl.getPanelCount());
+        ((TextField) find("#boardnameinput")).setText("empty");
```

Comment:
@dariusf can this be abstracted into a utility method somewhere (as a separate issue)? e.g. `setTextField(fieldName, text)`